Dockerfile

```diff
@@ -4,6 +4,7 @@ FROM ghcr.io/acorn-io/images-mirror/tonistiigi/binfmt:qemu-v6.2.0 AS binfmt
 FROM ghcr.io/acorn-io/images-mirror/moby/buildkit:v0.11.6 AS buildkit
 FROM ghcr.io/acorn-io/images-mirror/registry:2.8.1 AS registry
 FROM ghcr.io/acorn-io/images-mirror/rancher/klipper-lb:v0.3.5 AS klipper-lb
+FROM docker.io/thedadams/sleep:five AS sleep
```

Comment:
This needs to be mirrored, at least.

Reply:
Should we be maintaining a GitHub repo and build action for this image? That's assuming we're building this sleep binary from source somewhere.

Reply:
I made a repo for this.

Reply:
Thank you!